qiskit_experiments/framework/composite/parallel_experiment.py

```diff
@@ -54,20 +55,25 @@ def circuits(self):
 
         sub_circuits = []
         sub_qubits = []
+        sub_maps = []
         sub_size = []
         num_qubits = 0
 
         # Generate data for combination
-        for expr in self._experiments:
+        for sub_exp in self._experiments:
+
+            # Generate transpiled subcircuits
+            circuits = sub_exp._transpiled_circuits()
+
             # Add subcircuits
-            circs = expr.circuits()
-            sub_circuits.append(circs)
-            sub_size.append(len(circs))
+            sub_circuits.append(circuits)
+            sub_size.append(len(circuits))
 
             # Add sub qubits
-            qubits = list(range(num_qubits, num_qubits + expr.num_qubits))
+            qubits = list(range(num_qubits, num_qubits + sub_exp.num_qubits))
             sub_qubits.append(qubits)
-            num_qubits += expr.num_qubits
+            sub_maps.append({q: qubits[i] for i, q in enumerate(sub_exp.physical_qubits)})
```

Comment:
I find it impossible to follow the code in this method. Inline documentation will be very helpful.